TrainingWorkshop/ClassesAndInterfaces/Task 3/Person.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ClassesAndInterfaces.Task_3
+{
+    public class Person
+    {
+        List<Child> _Children = new List<Child>();
+
+        /// <summary>
+        /// Adds the child.
+        /// </summary>
+        public void AddChild(string name, int age)
+        {
+            _Children.Add(new Child() { Name = name, Age = age });
+        }
+
+        /// <summary>
+        /// Gets the children count.
+        /// </summary>
+        public int GetChildrenCount()
+        {
+            return _Children.Count;
+        }
+
+        class Child
```

Comment:
This is the example how nested class works, that's why class Child is situated inside of class Person